support-payment-api/src/test/scala/backend/PaypalBackendSpec.scala

```diff
@@ -341,6 +341,17 @@ class PaypalBackendSpec extends AnyWordSpec with Matchers with FutureEitherValue
 
         verify(mockSoftOptInsService, times(1)).sendMessage(any(), any())(any())
       }
+
+      "return error if email address has a comma" in new PaypalBackendFixture {
```

Comment:
Just checking my understanding - a comma infers a list of emails? if it does, should we add that somewhere in the test?

Reply:
I don't know what a comma is supposed to mean in this context, it's just an invalid email address.  A single contributor can only use one email to contribute.